app-tests/run_retry.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Reduce flaky tests by retrying them
+
+MAX_RETRIES=5
+RETRY_COUNT=0
+
+while [ $RETRY_COUNT -lt $MAX_RETRIES ]; do
+  echo "Running test (attempt $((RETRY_COUNT+1)) of $MAX_RETRIES)..."
```

Comment:
don't knwo if printf support here but that little more "clean"
printf "Running test (attempt %d of %d)...\n" $((++RETRY_COUNT)) $MAX_RETRIES